vite.config.ts

```diff
@@ -4,7 +4,7 @@ import { defineConfig } from "vitest/config";
 export default defineConfig({
   root: "./",
   test: {
-    workspace: ["packages/*", "tests/browser-integration"],
+    workspace: ["packages/*"],
```

Comment:
This is going to exclude the integration tests from the CI.

I also like that the integration tests runs when I type `pnpm test` on the root, because they help alot on catching regressions.

Why did you remove that?

Reply:
As I understand it Vite treats the contents of the folder as a src package and that breaks Turbo's test dependency resolution. Vitest doesn't need it to be there to run the tests (`pnpm test` in the packge and `turbo test` at the root still run the tests)

Reply:
Welp, we don't use `turbo test` in the unit-test workflow... Investigating.

Reply:
Reverting for now